simplified-app-shared/src/main/java/org/nypl/simplified/app/reader/ReaderActivity.java

```diff
@@ -760,13 +760,6 @@ public void onReadiumFunctionInitialize() {
 
     p.setRootUrls(hs.getURIBase().toString(), null);
 
-    LOG.debug("onReadiumFunctionInitialize: scheduling font injection");
-    // is this correct? inject fonts before book opens or after
-    UIThread.runOnUIThreadDelayed(() -> {
-      LOG.debug("onReadiumFunctionInitialize: injecting fonts now");
-      this.readium_js_api.injectFonts();
```

Comment:
This is not needed anymore? Did this ever work?

Reply:
It was actually your old patch that removed the `injectFonts()` method. The code seems to work without it (OpenDyslexic is usable here).

Reply:
Ah right, I guess I missed this call for the function.